rad-antonyms/counterfitting.py

```diff
@@ -0,0 +1,292 @@
+import configparser
+import random
+import sys
+import numpy as np
+from copy import deepcopy
+from datetime import datetime
+import tools
+
+
+class SettingConfig:
+    """
+    Class that encapsulates all the parameters and variables required for a counterfitting run
+    """
+
+    def __init__(self, config_path):
+
+        # Read the config file
+        self.config = configparser.RawConfigParser()
+        try:
+            self.config.read(config_path)
+        except OSError:
+            print("Unable to read config, aborting.")
+            return
+
+        # Read the word vectors path from the config
+        input_vectors_path = self.config.get("paths", "VEC_PATH")
+
+        # Read the vocabulary mode (all words or just dataset words) from the config
+        vocab_mode = self.config.get("settings", "VOCABULARY")
+        if vocab_mode == 'all':
+            vocab_path = self.config.get("paths", "VOCAB_PATH")
+        elif vocab_mode == 'small':
+            vocab_path = self.config.get("paths", "DATASET_VOCAB_PATH")
+        else:
+            print('Wrong value for parameter VOCABULARY in config. Exiting')
+            return
+
+        # Read paths of the files containing synonym and antonym pairs from the config
+        synonyms_path = self.config.get("paths", "SYNONYMS_PATH")
+        antonyms_path = self.config.get("paths", "ANTONYMS_PATH")
+
+        # Read and parse the mode (whether to include synonyms, antonyms or VSP pairs in the current run)
+        mode = self.config.get("settings", "MODE").replace("[", "").replace("]", "").replace(" ", "").split(",")
```

Comment:
Can you add a README file in this folder, in which you explain how to set the config file for each case? Just the properties and values that we care about (i.e. mode and vocab_mode). So that we have a documentation of what each value represents and how it should be set.

Reply:
Yes, the explanations were included in the omitted parameters.cfg. 
Will move them to a README and commit them both to this PR asap.